subprojects/common/sentry/src/main/kotlin/com/avito/android/sentry/SentryConfig.kt

```diff
@@ -44,7 +44,7 @@ sealed class SentryConfig : Serializable {
         val dsn: String,
         val environment: String,
         val serverName: String,
-        val release: String,
+        val release: String?,
```

Comment:
Can't find why it became nullable

Reply:
In Sentry, release is optional. I see no reasons to make it required in config.
It forces us to know about two states:
- No release
- "unknown" release